### PR TITLE
Release 2.25.1

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.25.1 / 2021-03-13
+
+* [BUGFIX] Fix a crash in `promtool` when a subquery with default resolution is used. #8569
+* [BUGFIX] Fix a bug that could return duplicate of datapoints in /api/v1/query. #8591
```
Comment (beorn7) [inline CHANGELOG.md:4]: Nit: Remove the "of". Or even more compact: "Fix possible return of duplicate datapoints in /api/v1/query."

**Review by beorn7 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (roidelapluie) [reply CHANGELOG.md]: thanks, applied.

**Review by dgl — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## 2.25.1 / 2021-03-14
+
+* [BUGFIX] Fix a crash in `promtool` when a subquery with default resolution is used. #8569
+* [BUGFIX] Fix a bug that could return duplicate datapoints in /api/v1/query. #8591
```
Comment (juliusv) [inline CHANGELOG.md:4]: Also in `/api/v1/query_range`, no? Maybe just say "in queries"?

**Comment by juliusv** [comment]
:+1: besides comment

**Pull request merged**